validate-date.js

```diff
@@ -1,31 +1,50 @@
-function validateDate(dateValue) {
-    if (dateValue == null) {
-        return 'Invalid Format';
-    }
-    return dateValidator(dateValue);
+function validateDate(dateValue, boolean = false) {
```

Comment:
Firstly.. nice way to handle both the newer and older approaches. This is good 💯 

One change that I'd suggest is about the "boolean" variable name. Can we rename that to something like "booleanResponse" or similar. Because it might confuse someone new with the well-known "boolean" datatype.

For future- I am thinking would there be any scenario where we might want to send a different type of response. If some day we end-up sending a new correct date suggestion(based on the typed in date) as response, in that case we might be sending a "date" object as a response. But let's leave this for now. Just sharing my thoughts. It might not even be necessary.

Reply:
Maybe we can change the parameter to **responseType**, and have, for now, string and boolean as accepted options. Later any other type can be added

Reply:
I vote for this. That'd be great IMO 💯

Reply:
will do!